app/dao/provider_details_dao.py

```diff
@@ -34,57 +33,92 @@ def dao_get_provider_versions(provider_id):
     ).all()
 
 
-@transactional
-def dao_reduce_sms_provider_priority(identifier, *, time_threshold):
-    """
-    Will reduce a chosen sms provider's priority, and increase the other provider's priority by 10 points each.
-    If either provider has been updated in the last `time_threshold`, then it won't take any action.
+def _adjust_provider_priority(provider, new_priority):
+    print('Adjusting provider priority - {} going from {} to {}'.format(
```

Comment:
Do you want to make this a log statement or get rid?

Reply:
whoops good spot, that should be a logger statement - i'd just changed it to help test